projects/vdk-core/src/vdk/internal/builtin_plugins/config/vdk_config.py

```diff
@@ -113,6 +114,15 @@ def vdk_configure(self, config_builder: ConfigurationBuilder) -> None:
             "Data Job execution can run a Data Job one or more times."
             "Each distinct run would a single attempt.",
         )
+        config_builder.add(
+            WRITE_DIRECTORY,
+            "/var/tmp",
```

Comment:
How does the user workflow looks for local executions  vs cloud execution?

```
def run(job_input): 
  job_input.get_write_directory().open("foo.txt").write("hello") 
```

There is a difference in local if foo.txt exists, the open would fail (unless override is allowed) on cloud there would not be foo.txt. 


We either need to document this difference or remove it. 

Are there any other differences that I am not thinking of right now.